apps/web/src/pages/Pools/EditPoolPage/components/PoolNameSection/utils.ts

```diff
@@ -85,13 +86,12 @@ export const getClassificationOptions = (
   classifications: Maybe<Classification>[],
   intl: IntlShape,
 ): Option[] => {
-  return classifications
-    .filter(notEmpty)
-    .map(({ id, group, level, name }) => ({
+  return alphaSortOptions(
+    classifications.filter(notEmpty).map(({ id, group, level, name }) => ({
```

Comment:
I think this is another redunancy

Reply:
Fixed by [f44d52d](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9534/commits/f44d52d8513c01020f725bf092de88c08aa4067f)